sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/GenerateOrdering.scala

```diff
@@ -69,7 +69,9 @@ object GenerateOrdering extends CodeGenerator[Seq[SortOrder], Ordering[InternalR
           }
       """
     }.mkString("\n")
-
+    val mutableStates = ctx.mutableStates.map { case (jt, name, init) =>
```

Comment:
@davies just saw your comment - what did you mean by ordering should be stateless?